src/reducers/setStudentId.test.js

```diff
@@ -0,0 +1,7 @@
+import { setStudentId } from './setStudentId'
+
+describe('setStudentId', () => {
+  it('should return initial state', () => {
```

Comment:
we should expand on this

Reply:
Yea I wasn't sure what this needed to actually be so I left this one for now.